corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -533,18 +533,12 @@ def all_case_properties_by_domain(domain, case_types=None, include_parent_proper
             defaults=('name',), include_parent_properties=include_parent_properties)
 
         for case_type, properties in six.iteritems(property_map):
-            if case_type in result:
-                result[case_type].extend(properties)
-            else:
-                result[case_type] = properties
-
-    cleaned_result = {}
-    for case_type, properties in six.iteritems(result):
-        properties = list(set(properties))
-        properties.sort()
-        cleaned_result[case_type] = properties
+            result[case_type].update(properties)
 
-    return cleaned_result
+    return {
+        case_type: sorted(properties)
+        for case_type, properties in result.items()
+    }
```

Comment:
Minor cleanup to help me make sense of this code.  Not that related to the rest of the PR.